roles/convert2rhel/README.md

```diff
@@ -0,0 +1,41 @@
+theforeman.foreman.convert2rhel
+===========================
+
+This role creates a basic configuration for everything needed to register and convert CentOS clients to Red hat Enterprise Linux.
+
+First is upload of manifest & sync of RHEL 7 & RHEL 8 repositories, for more detail see [content_rhel Role](https://github.com/theforeman/foreman-ansible-modules/blob/develop/roles/content_rhel/README.md).
```

Comment:
I can't exactly parse that sentence. Did you want to say "First it uploads a manifest and syncs RHEL, …"?